src/libhirte/common/cfg.c

```diff
@@ -112,11 +112,11 @@ int cfg_load_complete_configuration(
         if (custom_config_file != NULL) {
                 result = cfg_load_from_file(config, custom_config_file);
                 if (result < 0) {
-                        fprintf(stderr,
-                                "Error loading custom configuration file '%s': '%s'.\n",
-                                custom_config_file,
-                                strerror(-result));
                         if (result != -ENOENT) {
```

Comment:
Wouldn't it be better to unify those if statements into a signle one?

Reply:
Good point. Changed it.